common/meta/api/src/kv_api_test_suite.rs

```diff
@@ -532,6 +547,347 @@ impl KVApiTestSuite {
+                k2,
+                MatchSeq::Any,
+                Operation::Update(val2.clone()),
+                None,
+            ))
+            .await?;
+
+            // transaction by k1 and k2 condition
+            let txn_key1 = k1.to_string();
+            let txn_key2 = k2.to_string();
+
+            let condition = vec![
+                TxnCondition {
+                    key: txn_key1.clone(),
+                    expected: ConditionResult::Gt as i32,
+                    target_union: Some(txn_condition::TargetUnion::Seq(0)),
+                },
+                TxnCondition {
+                    key: txn_key2.clone(),
+                    expected: ConditionResult::Gt as i32,
+                    target_union: Some(txn_condition::TargetUnion::Seq(0)),
+                },
+            ];
+
+            let if_then: Vec<TxnOp> = vec![
+                // change k1
+                TxnOp {
+                    request: Some(txn_op::Request::Put(TxnPutRequest {
+                        key: txn_key1.clone(),
+                        value: val1_new.to_vec(),
+                        prev_value: true,
+                    })),
+                },
+                // change k2
+                TxnOp {
+                    request: Some(txn_op::Request::Put(TxnPutRequest {
+                        key: txn_key2.clone(),
+                        value: b"new_v2".to_vec(),
+                        prev_value: true,
+                    })),
+                },
+                // get k1
+                TxnOp {
+                    request: Some(txn_op::Request::Get(TxnGetRequest {
+                        key: txn_key1.clone(),
+                    })),
+                },
+                // delete k1
+                TxnOp {
+                    request: Some(txn_op::Request::Delete(TxnDeleteRequest {
+                        key: txn_key1.clone(),
+                        prev_value: true,
+                    })),
+                },
+                // get k1
+                TxnOp {
+                    request: Some(txn_op::Request::Get(TxnGetRequest {
+                        key: txn_key1.clone(),
+                    })),
+                },
+            ];
+
+            let else_then: Vec<TxnOp> = vec![];
+            let txn = TxnRequest {
+                condition,
+                if_then,
+                else_then,
+            };
+
+            let resp = kv.transaction(txn).await?;
+
+            assert!(resp.success);
+
+            let expected: Vec<TxnOpResponse> = vec![
+                // change k1
+                TxnOpResponse {
+                    response: Some(txn_op_response::Response::Put(TxnPutResponse {
+                        key: txn_key1.clone(),
+                        prev_value: convert_seqv_to_pb(Some(SeqV::with_meta(
+                            4,
+                            None,
+                            val1.clone(),
+                        ))),
```

Comment:
There is a method `new()`, which does not need `meta`:

If you'd like to save several lines of code:D

```rust
impl<T> SeqV<T> {
    pub fn new(seq: u64, data: T) -> Self {
        Self {
            seq,
            meta: None,
            data,
        }
    }

    pub fn with_meta(seq: u64, meta: Option<KVMeta>, data: T) -> Self {
        Self { seq, meta, data }
    }
```